server/database/config.js

```diff
@@ -12,18 +12,33 @@ var orm = new Sequelize(dbconfig.database, dbconfig.username, dbconfig.password,
 
 var Messages = orm.define('messages', {
   message: Sequelize.TEXT,
-  name: Sequelize.STRING
+  name: Sequelize.STRING,
+  groupID: Sequelize.STRING
 });
 
 var Users = orm.define('users', {
-  name: Sequelize.STRING
+  name: Sequelize.STRING,
+  password: Sequelize.STRING,
+  groupID: Sequelize.STRING
+});
+
+var Groups = orm.define('groups', {
+  groupID: Sequelize.STRING
```

Comment:
I don't like how groupID is a string, which implies it should be the name of the group, yet ID is used as a primary key for other tables as a number to reference other things. This is weird to me.